scripts/nayduck.py

```diff
@@ -67,13 +70,28 @@ def _parse_args():
     return args
 
 
-def get_curent_sha():
-    return subprocess.check_output(['git', 'rev-parse', 'HEAD'], text=True)
+def get_sha(branch: str):
+    try:
+        sha = subprocess.check_output(['git', 'rev-parse', branch], text=True)
+
+    except subprocess.CalledProcessError as e:
+        remote_branch = 'remotes/origin/' + branch
+        print(
+            f"Couldn't find a local branch \'{branch}\'. Trying remote: {remote_branch}"
```

Comment:
Can you also print the stdout and stderr of the failed check_output call - unless it already works like that then ignore me :)

Reply:
Yes, it does indeed, see the test above for "./scripts/nayduck.py  -t local_tests.txt -b waclaw-cold-tracked-shards "